modules/ui/sources/window.es

```diff
@@ -121,6 +121,9 @@ export default class {
 
     }.bind(this));
 
+    //mock default FF function
+    this.popup.enableOneOffSearches = function() {}
```

Comment:
Ah, OK, we may want to allow this to work in the future, I'll talk to the team about it. Thanks!